backend/app/controllers/api/v1/users/project_developers_controller.rb

```diff
@@ -0,0 +1,34 @@
+module API
+  module V1
+    module Users
+      class ProjectDevelopersController < BaseController
+        skip_before_action :require_json!, only: :create
+        before_action :authenticate_user!
+
+        def create
+          current_user.with_lock do
+            raise API::UnprocessableEntityError, I18n.t("errors.messages.user_multiple_accounts") if current_user.account_id.present?
+
+            account = Account.create! account_params
```

Comment:
missing set `owner_id` to current_user.id. Account model is missing `has_one :owner`

Reply:
Added missing owner, thanks @tsubik ;)